bot/cogs/snakes.py

```diff
@@ -15,31 +43,99 @@ class Snakes:
     def __init__(self, bot: AutoShardedBot):
         self.bot = bot
 
-    async def get_snek(self, name: str = None) -> Dict[str, Any]:
-        """
-        Go online and fetch information about a snake
+    @staticmethod
+    def parse_kwarg(kwarg: str):
+        """Return the value of a kwarg (needs manual assignment)"""
+        try:
+            items = kwarg.split("=")
+            value = items[1]
+        except (IndexError, AttributeError):
+            value = kwarg
+
+        try:
+            parsed = ast.literal_eval(value)
+        except (NameError, ValueError):
+            parsed = value
+
+        return parsed
+
+    @staticmethod
+    def snake_url(name: str):
+        """Get the URL of a snake"""
+
+        def format_url(text: str):
+            """Get the full URL with that snake :D"""
+            return BASEURL.format(urllib.parse.quote_plus(text))
+
+        # Check if the snake name is valid
+        if name.upper() in [name.upper() for name in SNAKES]:
```

Comment:
Instead of `upper()`ing all the names every time you do this, maybe they could be stored that way?